Makefile

```diff
@@ -16,17 +16,36 @@
 #   - convenience: redirects targets to ./vmdk_plugin so 'make'
 #                  can be run from the top
 #   - drone invariant: defines deb/rpm/build/testasroot targets so drone.yml
-#                   can be kept as-is 
+#                   can be kept as-is
 
 TARGETS := all \
 	deploy-all deploy-esx deploy-vm deploy-vm-test deploy\
 	test-all test-vm test-esx testremote testasroot\
-	clean-all clean clean-vm clean-esx \
-	build-all build dockerbuild \
+	clean-all clean-vm clean-esx \
+	build-all dockerbuild \
 	deb rpm package gvt documentation
 
-# default target
-default: build-all
+BUILD := ./misc/scripts/build.sh
+
+# default target, build ui then build vib, rpm, deb
+default: dockerized-build-ui build-all
```

Comment:
We should not include UI build in default make and should not include UI files in VIB until feature is complete and ready to be released. Until then, please have separate build target for UI and vib.